src/elements/Header/_Header.js

```diff
@@ -1,42 +1,98 @@
 import cx from 'classnames'
-import React, { createElement, Component, PropTypes } from 'react'
-
+import React, { PropTypes } from 'react'
 import META from '../../utils/Meta'
+import {
+  getUnhandledProps,
+  iconPropRenderer,
+  imagePropRenderer,
+  useKeyOnly,
+} from '../../utils/propUtils'
+
+function _Header(props) {
+  const {
+    _sdClass, _headerElement,
+    left, center, right, justified, dividing, block, attached, floating,
+    icon, image, children, className,
+  } = props
+
+  const classes = cx(
```

Comment:
Let's add support for [`disabled`](http://semantic-ui.com/elements/header#disabled) and [`color`](http://semantic-ui.com/elements/header#colored):

```js
<Header disabled />

cx(
  useKeyOnly(disabled, 'disabled')
)
```

```js
<Header color='red' />

cx(
  color,  // since only the value is used, there is no helper method
)
```